services/catarse/app/views/catarse_bootstrap/user_notifier/mailer/new_followers_subject.text.slim

```diff
@@ -1,4 +1,5 @@
--if @notification.user.followers_since_last_day.count == 1
+- followers_count = @notification.metadata['followers'].present? ? @notification.metadata['followers'].count : 0
```

Comment:
**Sugestão:**
Se for o `metadata['user_follow_ids']` for `nil`, ao fazer o `.to_a` ele transforma em `[]`, e o `.size` vai retornar zero.
```suggestion
- followers_count = @notification.metadata['user_follow_ids'].to_a.size
```